tgui/packages/tgui/interfaces/VotePanel.tsx

```diff
@@ -26,12 +26,15 @@ type ActiveVote = {
   question: string | null;
   timeRemaining: number;
   choices: Option[];
+  countMethod: number;
```

Comment:
Because this is TypeScript, use an enum with `= 1` and `= 2` so that we can actually read the values of this

Reply:
Alrighty!